src/ui/components/LeagueTopBar.tsx

```diff
@@ -62,6 +65,37 @@ const LeagueTopBar = memo(() => {
 		};
 	}, [updateNumberOfScoreBoxes]);
 
+	useEffect(() => {
+		const leagueTopBar = leagueTopBarRef.current;
+		if (
+			!leagueTopBar ||
+			!show ||
+			leagueTopBar.scrollWidth <= leagueTopBar.clientWidth
+		)
+			return;
+
+		console.log("added");
+		leagueTopBar.addEventListener("wheel", handleWheel, { passive: false });
+
+		return () => {
+			console.log("removed");
+			leagueTopBar.removeEventListener("wheel", handleWheel, {
+				passive: false,
+			});
+		};
+	});
+
+	const handleWheel = useCallback(e => {
+		e.preventDefault();
+		const leagueTopBarPosition = leagueTopBarRef.current.scrollLeft;
+
+		leagueTopBarRef.current.scrollTo({
+			top: 0,
+			left: leagueTopBarPosition - e.deltaY + e.deltaX,
+			behaviour: "smooth",
+		});
+	}, []);
+
```

Comment:
Enables scrolling for league-top-bar. 

- I added the event listener as  `e.preventDefault()` will not work with the react `onwheel` event without `leagueTopBar.addEventListener("wheel", handleWheel, { passive: false });`.

- My one conern is that the `useEffect()` will run after every render, which may create multiple "wheel" event listeners. I wasn't actually to create this issue, but i left the two `console.log()` in for you to easily test if you can create the issue.

Reply:
> My one conern is that the useEffect() will run after every render, which may create multiple "wheel" event listeners. I wasn't actually to create this issue, but i left the two console.log() in for you to easily test if you can create the issue.

If you add a dependency array (2nd arg) of `[handleWheel, show]` to that useEffect, it will run many fewer times.

Reply:
Added